docs/sources/concepts/configuration-syntax/_index.md

```diff
@@ -8,51 +8,81 @@ weight: 10
 
-The {{< param "PRODUCT_NAME" >}} syntax aims to reduce errors in configuration files by making configurations easier to read and write.
-{{< param "PRODUCT_NAME" >}} configurations use blocks that can be easily copied and pasted from the documentation to help you get started as quickly as possible.
+{{< figure src="/media/docs/alloy/flow-diagram-small-alloy.png" alt="Alloy flow diagram" >}}
 
-An {{< param "PRODUCT_NAME" >}} configuration file tells {{< param "PRODUCT_NAME" >}} which components to launch and how to bind them together into a pipeline.
-
-The {{< param "PRODUCT_NAME" >}} syntax uses blocks, attributes, and expressions.
+Here is a simple example of a fully worked configuration, showing the basic concepts, and how an Alloy config comes together into a pipeline.
 
 ```alloy
-// Create a local.file component labeled my_file.
-// This can be referenced by other components as local.file.my_file.
-local.file "my_file" {
-  filename = "/tmp/my-file.txt"
+// Collection: mount a local directory with a certain path spec
+local.file_match "applogs" {
+    path_targets = [{"__path__" = "/tmp/app-logs/app.log"}]
+}
+
+// Collection: Take the file match as input, and scrape those mounted log files
+loki.source.file "local_files" {
+    targets    = local.file_match.applogs.targets
+
+    // This specifies which component should process the logs next, the "link in the chain"
+    forward_to = [loki.process.add_new_label.receiver]
 }
 
-// Pattern for creating a labeled block, which the above block follows:
-BLOCK_NAME "BLOCK_LABEL" {
-  // Block body
-  IDENTIFIER = EXPRESSION // Attribute
+// Transformation: pull some data out of the log message, and turn it into a label
+loki.process "add_new_label" {
+    stage.logfmt {
+        mapping = {
+            "extracted_level" = "level",
+        }
+    }
+
+    // Add the value of "extracted_level" from the extracted map as a "level" label
+    stage.labels {
+        values = {
+            "level" = "extracted_level",
+        }
+    }
+
+    // The next link in the chain is the local_loki "receiver" (receives the telemetry)
+    forward_to = [loki.write.local_loki.receiver]
 }
 
-// Pattern for creating an unlabeled block:
-BLOCK_NAME {
-  // Block body
-  IDENTIFIER = EXPRESSION // Attribute
+// Anything that comes into this component gets written to the loki remote API
+loki.write "local_loki" {
+    endpoint {
+        url = "http://loki:3100/loki/api/v1/push"
+    }
 }
 ```
 
-{{< param "PRODUCT_NAME" >}} is designed with the following requirements in mind:
+The {{< param "PRODUCT_NAME" >}} syntax aims to reduce errors in configuration files by making configurations easier to read and write.
+{{< param "PRODUCT_NAME" >}} configurations use blocks that can be easily copied and pasted from the documentation to help you get started as quickly as possible.
```

Comment:
```suggestion
The {{< param "PRODUCT_NAME" >}} syntax aims to reduce errors in configuration files by making configurations easier to read and write.
The {{< param "PRODUCT_NAME" >}} syntax uses blocks, attributes, and expressions.
The blocks can be copied and pasted from the documentation to help you get started as quickly as possible.
```

The sentence about blocks, attributes, and expressions was out of place in the flow. I moved it up here, and suggest deleting it from line 64. We might need to look into how lines 61 and 62 fit into the flow of ideas here too...